network/load-balancer/how-to/use-with-private-network.mdx

```diff
@@ -13,7 +13,9 @@ categories:
   - network
 ---
 
-The Private Network feature of the managed Load Balancers product lets you configure your Load Balancer with a Private Network. [Scaleway Private Networks](/network/vpc/concepts/#private-networks) allow your cloud resources to communicate in an isolated and secure network. Load Balancers using Private Networks can communicate with your backend servers without using the public Internet. You can add your Load Balancer to one or multiple existing Private Networks. The feature supports both static IP and DHCP configuration for the Load Balancer IP in the Private Network.
+The Private Network feature of the managed Load Balancers product lets you attach your Load Balancer to up to eight different Private Networks. [Scaleway Private Networks](/network/vpc/concepts/#private-networks) allow your cloud resources to communicate in an isolated and secure network. Load Balancers using Private Networks can communicate with their backend servers without using the public Internet. The feature supports both static IP and DHCP configuration for the Load Balancer IP in each Private Network.
```

Comment:
> The feature supports both static IP and DHCP configuration for the Load Balancer IP in each Private Network.

@RoRoJ in the case of a private LB you won't have the choice, it will only be via DHCP, so you need to make sure DHCP is enabled in the private network. If it's not the case the Console won't allow you to proceed with attachment. Public LB will also move to this method very soon, so maybe we can harmonize the documentation ahead of time. In the end you will just "attach to a private network" you won't be choosing an attachment mode.